packages/editor/src/plugins/dropzone-image/editor.tsx

```diff
@@ -71,6 +69,17 @@ export function DropzoneImageEditor(props: DropzoneImageProps) {
     duplicateAnswerZone,
   } = useAnswerZones(answerZones)
 
+  const handleChangeImageButtonClick = useCallback(() => {
+    if (!backgroundImage.defined) return
+
+    dispatch(
+      runChangeDocumentSaga({
+        id: backgroundImage.id,
+        state: { initial: (curr) => ({ ...(curr as object), src: '' }) },
+      })
```

Comment:
Redo doesn't work for me, when I click the button, then undo, and then try to redo. My feeling tells me that it could be because we're missing `state.executor` and/or `reverse` props in the object passed to the saga. If you think that redo is a crucial feature here, we could pair and try to figure it out together.

Reply:
Thank you. Nice catch! 
I'd merge it for now, but would like a session to look into it when the other PRs are merged (so I we don't create conflicts here)